apps/storybook/stories/Namekit/DisplayedPrice.stories.tsx

```diff
@@ -0,0 +1,299 @@
+import { Currency } from "@namehash/ens-utils";
+import {
+  AltPriceDisplayFormat,
+  DisplayedPrice,
+  PriceDisplayPosition,
+  PriceDisplaySize,
+  PriceSymbolPosition,
+  PriceSymbology,
+} from "@namehash/namekit-react";
+import type { Meta, StoryObj } from "@storybook/react";
+
+const meta: Meta<typeof DisplayedPrice> = {
```

Comment:
Can we please make it so that Storybook is better configured to work with our types and the overall type system?

For example, how complicated is it to make it so that the controls for these are typed? For example, currency as a dropdown where only valid currency options can be input. Etc.. If this is complicated please let me know.

![CleanShot 2024-06-16 at 13 05 10](https://github.com/namehash/nameguard/assets/126201998/345dbcfa-528e-4d66-8334-90a9e49c519d)

Reply:
@lightwalker-eth I manage to improve a lot the props typing but the object props could not be set different